uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -44,10 +46,13 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+
 public class BrokerHost extends AbstractTestBroker {
     private final static String TAG = BrokerHost.class.getSimpleName();
     // tenant id where lab api and key vault api is registered
     private final static String LAB_API_TENANT_ID = "72f988bf-86f1-41af-91ab-2d7cd011db47";
+    // flight to enable/disable the multiple wpj feature
+    private final static String FLIGHT_FOR_WORKPLACE_JOIN_CONTROLLER = "ENABLE_MULTIPLE_WORKPLACE_JOIN_PP";
```

Comment:
can we use the enum directly?

Reply:
I am hesitant to do this change because it will require to make uiautomationutilities dependent on broker4j, if you think this is fine I can do the change and update the pipelines that build uiautomationutilities